packages/swift/build.sh

```diff
@@ -12,6 +12,12 @@ TERMUX_PKG_DEPENDS="binutils-gold, clang, libc++, ndk-sysroot, libandroid-glob,
 TERMUX_PKG_BUILD_DEPENDS="cmake, ninja, perl, pkg-config, rsync"
 TERMUX_PKG_BLACKLISTED_ARCHES="i686"
 TERMUX_PKG_NO_STATICSPLIT=true
+# Build of swift uses cmake, but the standard
+# termux_step_configure_cmake function is not used. Instead we set
+# TERMUX_PKG_FORCE_CMAKE to make the build system aware that cmake is
+# needed.
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_CMAKE_BUILD=Ninja
```

Comment:
I have those two setup in the host build, which has been good enough.

Reply:
Yeah, but then termux_setup_{cmake,ninja} are not run if `./build-package.sh swift` is run twice in a row without deleting the hostbuild dir, i.e. if the hostbuild step is not run. libllvm has the same issue

Reply:
Sure, I just haven't had a problem with that. I guess you're worried about [the local install not being added to the path](https://github.com/termux/termux-packages/blob/master/scripts/build/setup/termux_setup_cmake.sh#L26)? I have recent versions installed in Fedora anyway, but maybe that breaks on Ubuntu.

Reply:
Yes, in the docker image cmake and ninja are not in PATH if the setup functions are not run